src/sql/parts/notebook/notebookUtils.ts

```diff
@@ -116,4 +116,10 @@ export function tryMatchCellMagic(input: string): string {
 	let match = magicRegex.exec(firstLine);
 	let magicName = match && match[1];
 	return magicName;
+}
+
+export async function asyncForEach(array, callback) {
```

Comment:
this needs a return type

Reply:
is it a coding standard?

Reply:
Yes it is, please add return type for all methods. is it a promise? I wouldn't know immediately. Similarly please type the inputs as it really isn't clear how all this works.

Reply:
We need to update our TS Lint rules to enforce more stuff like this but it's helpful for people calling the function or modifying this code in the future because otherwise the types will show up as `any` and it'll be harder to figure out what this code is doing